deploy/route-monitor-operator-manager-role.ClusterRole.yaml

```diff
@@ -130,6 +130,16 @@ rules:
       - get
       - patch
       - update
+  - apiGroups:
```

Comment:
Can remove change in favor of #205

Reply:
See [above](https://github.com/openshift/route-monitor-operator/pull/204#discussion_r1164738886)